Makefile

```diff
@@ -139,6 +139,10 @@ logs-clean:
 	$(info "Removing pytest worker logs...")
 	-rm -f robottelo_gw*.log
 
+clean-shared:
```

Comment:
It is missing on `PHONY` list at https://github.com/SatelliteQE/robottelo/blob/6.2.z/Makefile#L172